components/supervisor/frontend/src/index.ts

```diff
@@ -10,9 +10,37 @@
  */
 import { IDEMetricsServiceClient, MetricsName } from "./ide/ide-metrics-service-client";
 IDEMetricsServiceClient.addCounter(MetricsName.SupervisorFrontendClientTotal).catch(() => {})
-window.addEventListener('error', () => {
-    IDEMetricsServiceClient.addCounter(MetricsName.SupervisorFrontendErrorTotal).catch(() => {})
-})
+
+declare global {
+    interface Window {
+        gatheredErrors: Map<string, string>[];
```

Comment:
use gitpod in name of global variables, i.e. `gitpodIDEErrors`

Reply:
Thanks for the reminder :pray: